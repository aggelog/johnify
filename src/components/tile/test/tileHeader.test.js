import React from 'react';
import TileHeader from '../tileHeader';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<TileHeader title="title" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});