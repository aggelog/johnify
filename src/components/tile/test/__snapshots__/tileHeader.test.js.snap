// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tile-header__container"
>
  <div
    className="title-header__title"
  >
    title
  </div>
</div>
`;
